js/renovation/scheduler/appointment-tooltip/tooltip-item-layout.tsx

```diff
@@ -18,6 +18,9 @@ import TooltipItemContent from './tooltip-item-content';
 
 export const viewFunction = (viewModel: TooltipItemLayout) => {
   const useTemplate = !!viewModel.props.itemContentTemplate;
+  const formattedContent = viewModel.props.getTextAndFormatDate!(
```

Comment:
It's better to avoid any calculations in view function, component's getter is better here

Reply:
This calculation depends on value we get from getter (currentData getter). Would it be correct to move this to getters? Because if I call this getter inside another getter, it will be calculated twice: when it is called inside another getter and before it is passed to the viewFunction